### ISSUE TITLE
[Accessibility] Expense Report: 'ER-132' control is not accessible with keyboard using Tab key on Expense Report Pane.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6259]
### Target Platforms

Android

### SDK Version

Application Version: 2021.06.04.1

### Application Name

Visualizer

### Problem Description

[34055892](https://microsoft.visualstudio.com/OS/_workitems/edit/34055892)

'ER-132' control is not accessible with keyboard using Tab key on Expense Report Pane.

### Card JSON

[ExpenseReport.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.2/Scenarios/ExpenseReport.json)
